clang/unittests/Analysis/FlowSensitive/TypeErasedDataflowAnalysisTest.cpp

```diff
@@ -975,6 +994,35 @@ TEST_F(WideningTest, DistinctValuesWithSamePropertiesAreEquivalent) {
       });
 }
 
+TEST_F(WideningTest, DistinctValuesWithDifferentPropertiesWidenedToTop) {
+  std::string Code = R"(
+    void target(bool Cond) {
+      int *Foo;
+      int i = 0;
+      Foo = nullptr;
+      while (Cond) {
+        Foo = &i;
+      }
+      (void)0;
+      /*[[p]]*/
+    }
+  )";
+  runDataflow(
+      Code,
+      [](const llvm::StringMap<DataflowAnalysisState<NoopLattice>> &Results,
+         ASTContext &ASTCtx) {
+        ASSERT_THAT(Results.keys(), UnorderedElementsAre("p"));
```

Comment:
We do this in a lot of existing tests, but it's unnecessary: `getEnvironmentAtAnnotation()` itself asserts that the annotation exists.

```suggestion
```

Reply:
Great. I've removed it here and in all other cases of WideningTest.